src/mpcCoreKit.ts

```diff
@@ -1642,8 +1692,15 @@ export class Web3AuthMPCCoreKit implements ICoreKit, IMPCContext {
     }
   }
 
-  private async sign_frost(data: Buffer, keyTweak?: BN): Promise<Buffer> {
-    const nodeDetails = fetchLocalConfig(this.options.web3AuthNetwork, this.keyType, this._sigType);
+  private async sign_frost(args: {
+    data: Buffer;
+    keyTweak?: BN;
+    keyType: KeyType;
```

Comment:
you can derive the key type from the sig type

Reply:
updated